source/common/stats/utility.cc

```diff
@@ -19,6 +19,7 @@ std::string Utility::sanitizeStatsName(absl::string_view name) {
   std::string stats_name = std::string(name);
   std::replace(stats_name.begin(), stats_name.end(), ':', '_');
   std::replace(stats_name.begin(), stats_name.end(), '\0', '_');
+  std::replace(stats_name.begin(), stats_name.end(), '/', '_');
```

Comment:
Please update the tests in `test/common/stats/thread_local_store_test.cc` and `test/common/stats/isolated_store_impl_test.cc` that validate that `:` and `\0` are sanitized.

Reply:
No test is breaking but I am not concern if istio fine.
@douglas-reid Can you confirm? A runtime flag can be added if istio would be impacted

Reply:
The CI failure is `timed out waiting for cluster.xdstp_//test/envoy.config....`, I only change the stats name here, do not know the cause. Did I miss something import?

Reply:
@lambdai as far as i know, none of the istio envoy stats have `/`s in their _name_.  This should not impact Istio.

Reply:
@douglas-reid thanks for your confirmation.

Reply:
Thanks!